Bricks/strings/test.cc

```diff
@@ -642,9 +643,16 @@ TEST(TimeIntervalAsString, SmokeTest) {
 
 TEST(Util, Escape) {
   EXPECT_EQ("Smoke.", EscapeForCPlusPlus("Smoke."));
+  EXPECT_EQ("vertical|bar", EscapeForCPlusPlus("vertical|bar"));
   EXPECT_EQ("new\\nline", EscapeForCPlusPlus("new\nline"));
   EXPECT_EQ("t\\ts\\tv", EscapeForCPlusPlus("t\ts\tv"));
   EXPECT_EQ("Testing \\'quote\\' and \\\"quote\\\".", EscapeForCPlusPlus("Testing 'quote' and \"quote\"."));
```

Comment:
It's used to generate the `CURRENT_FIELD_DESCRIPTION(name, "blah")`, where `"blah"` should compile.